scalableminds/webknossos
# Improving the selection of the next active tree after tree deletion

This PR improves the selection for the next active tree after deleting a tree.
For further information: [Link to Discuss](https://discuss.webknossos.org/t/activate-next-tree-in-list-after-tree-deletion/1551)
### URL of deployed dev instance (used for testing):
- https://betteractivetreeaftertreedeletion.webknossos.xyz

### Steps to test:
- open a skeleton tracing
- create a few trees (can be empty)
- After the deletion of a tree in the middle of the tree list, the next active tree should be the tree above the deleted tree.
- After the deletion of the first tree in the tree list, the next active tree should the first in the list.
- After the deletion the last tree in the tree list, the next active tree should be the tree last tree in the list.

### Issues:
- fixes #4323

------
